### DISCUSSION TITLE
Configuration possibilty of dark windows mode and light default app mode

### DISCUSSION THREAD

**Discussion by schoesa** [AutoDarkMode/Windows-Auto-Night-Mode#857] [Q&A] [answered]
In Windows this combination is possibly under Personalizsation -> Colors. Can I change the config regarding the `Force Light Button` for the same behaviour?

![image](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/34265840/3b75395d-6fa9-48fc-9d82-ada1eb35f165)

**Comment by Armin2208** [comment, marked as answer]
![grafik](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/32982430/e01ba93e-046d-4dba-8b4b-2ac640624999)

**Comment by schoesa** [comment]
Thanks @Armin2208
